packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1711,8 +1712,35 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
                 return { ...attemptData, error };
             }
 
+            const summary = summarizeResult.summary.tree;
+            const summaryTree = summary[channelsTreeName];
+            let totalDataStoreCount = 0;
+            let unreferencedDataStoreCount = 0;
+            let handleCount = 0;
+            if (summaryTree.type === SummaryType.Tree) {
+                if (summaryTree.tree !== undefined) {
+                    totalDataStoreCount = Object.keys(summaryTree.tree).length;
+                    for(const key of Object.keys(summaryTree.tree)) {
+                        const dataStore = summaryTree.tree[key];
+                        if (dataStore.type === SummaryType.Tree && dataStore.unreferenced === true) {
+                            unreferencedDataStoreCount ++;
+                        }
+                        if (dataStore.type === SummaryType.Handle) {
+                            handleCount ++;
+                        }
+                    }
+                }
+            }
+
+            const topLevelSummaryStat: ITopLevelSummaryStats = {
+                ...summarizeResult.stats,
+                totalDataStoreCount,
+                unreferencedDataStoreCount,
+                totalSummarizedDataStoreCount: totalDataStoreCount - handleCount,
```

Comment:
I'd remove "total" from names - I do not think it adds value (and shorter names are more likely to be used somewhere else in telemetry, decreasing number of columns we have in Kusto)

dataStoreCount
summarizedDataStoreCount
unreferencedDataStoreCount

or maybe

dataStoreCount
summarizedDataStoreCount
summarizedUnreferencedDataStoreCount (to ensure it's clear that unreferenced is only for summarized data stores).